moto/rds3/models/db_cluster.py

```diff
@@ -37,6 +37,7 @@ def __init__(
         storage_encrypted=False,
         tags=None,
         vpc_security_group_ids=None,
+        deletion_protection=True,
```

Comment:
This should be `False` by default, right?  This would also obviate the need to add `DeletionProtection=False` to the tests below.

Reply:
Updated